build/__dotNetCoreSdkVersions.sh

```diff
@@ -6,7 +6,7 @@ DOT_NET_CORE_22_SDK_VERSION='2.2.207'
 DOT_NET_CORE_30_SDK_VERSION='3.0.103'
 DOT_NET_CORE_31_SDK_VERSION='3.1.415'
 DOT_NET_CORE_50_SDK_VERSION='5.0.302'
-DOT_NET_50_SDK_VERSION='5.0.403'
+DOT_NET_50_SDK_VERSION='5.0.406'
```

Comment:
wondering if we want to combine `DOT_NET_CORE_50_SDK_VERSION `and `DOT_NET_50_SDK_VERSION`? Since they're having same major and minor version as 5.0